functions/__fzf_search_current_dir.fish

```diff
@@ -2,10 +2,22 @@ function __fzf_search_current_dir --description "Search the current directory us
     # Make sure that fzf uses fish to execute __fzf_preview_file.
     # See similar comment in __fzf_search_shell_variables.fish.
     set --local --export SHELL (command --search fish)
-    set file_paths_selected (
-        fd --hidden --color=always --exclude=.git 2>/dev/null |
-        fzf --multi --ansi --preview='__fzf_preview_file {}' --query=(commandline --current-token)
-    )
+
+    set fd_arguments --hidden --color=always --exclude=.git
+    set fzf_arguments --multi --ansi --preview='__fzf_preview_file {}'
+    set token (commandline --current-token | string unescape)
```

Comment:
in what cases would you have to string unescape? if the current token is "folder name"?

Reply:
This can happen if the folder name has escaped whitespace or other special characters (e.g. braces) that come from fish TAB completion.